### DISCUSSION TITLE
Can we have the ability to disable Discovery?

### DISCUSSION THREAD

**Discussion by daemonspudguy** [kraxarn/spotify-qt#54] [Ideas]
Librespot (the library this program uses) supports it, and outright requires it on FreeBSD, or else it crashes. Can you implement a toggle for that?

**Comment by kraxarn** [comment]
What does it do exactly? Seems to work fine with it disabled, so a setting could be added.
Added in 8f6916283f1ca7fd412962faec37b849b925f652 and will be included in v3.4.
